packages/grafana-ui/src/components/ColorPicker/ColorSwatch.tsx

```diff
@@ -30,14 +30,13 @@ export const ColorSwatch = React.forwardRef<HTMLDivElement, Props>(
     const styles = getStyles(theme, variant, color, isFocusVisible, isSelected);
     const hasLabel = !label;
```

Comment:
Do we still need to add the other '!' here?

Reply:
Feel free to merge this PR, I can close mine :) thanks for digging into this and finding the issue!